tools/cli-bootstrap/repo-dependencies-installer.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python 
```

Comment:
Can we just remove all this stuff?  I don't think it's super worthwhile.  I don't know of another project that tries to automagically install a tool-chain on your behalf.  I also worry that we'll just constantly be adding packages for different things.  We should just document the toolchain at a high level.